Extend Pi-Hole documentation (#871)
MichaIng/DietPi-Docs: docs/software/dns_servers.md

    - [The Big Blocklist Collection by `WaLLy3K`](https://firebog.net/)
    - [Phishing Army blocklist](https://phishing.army/)
    - [Whitelist collection by `anudeepND`](https://github.com/anudeepND/whitelist)

>     - [RPiList](https://github.com/RPiList/specials/blob/master/Blocklisten.md): Block- and whitelists with focus on german domains

Reviewer: Also when used as adjectives, in English, countries are always starting with capital letter 🙂:
```suggestion
    - [RPiList](https://github.com/RPiList/specials/blob/master/Blocklisten.md): Block- and whitelists with focus on German domains
```

    - [The Big Blocklist Collection by `WaLLy3K`](https://firebog.net/)
    - [Phishing Army blocklist](https://phishing.army/)
    - [Whitelist collection by `anudeepND`](https://github.com/anudeepND/whitelist)
    - [RPiList](https://github.com/RPiList/specials/blob/master/Blocklisten.md): Block- and whitelists with focus on german domains

=== "Handling many lists"

    If you try to add many block- resp. whitelists (e.g. > 1 Mio), it can occur that the `/tmp` filesystem overflows.  

>     E.g., if `pihole -g` seems to fail with an error message like `sed: couldn't write 44 items to stdout: No space left on device`, then maybe `/tmp` is full.  
>     This can be checked using `df` command:

Reviewer: ```suggestion
    If `pihole -g` fails with an error message like `sed: couldn't write 44 items to stdout: No space left on device`, you can verify this case using the `df` command:
```

    ```
    root@dietpi:~# df -h /tmp
    Dateisystem    Größe Benutzt Verf. Verw% Eingehängt auf
    tmpfs           995M    995M     0  100% /tmp
    root@dietpi:~#
    ```

    The output value of "100 %" signals a full `/tmp` filesystem.

>     In this case the `/etc/fstab` could be changed to a larger `/tmp` by editing it. We propose to set it to a maximum value of 75 % of your RAM size (e.g. in case of 2 GB RAM, you could set it to 1500 M). 

Reviewer: ```suggestion
    In this case the `/etc/fstab` could be changed to a larger `/tmp` by editing it. We propose to set it to a maximum value of 75 % of your RAM size. E.g. in case of 2 GB RAM, you could adjust the mount option to `size=1500M`.
```
Or shall we show how the complete line could look like?
